src/main/proto/com/google/api/codegen/samplegen/v1/sample_config_v1.proto

```diff
@@ -24,22 +24,25 @@ message SampleSpec {
   // value in the emitted samples.
   string id = 1;
 
+  // The title of the sample.
+  string title = 2;
```

Comment:
We need `title` as well.

cc/ @vchudnov-g

Reply:
Yeah, I was wondering about that....

Reply:
(Remind me: is the title use anywhere other than metadata?)

Reply:
No. That's the only place.